### PR TITLE
[TR] PIM-2736 bis: Add comment in Notifications

### PR DESCRIPTION
**Pull request by willy-ahva**
| Q                 | A
| ----------------- | ---
| Specs             | ok
| Behats            |  ok
| Blue CI           | ok
| Changelog updated | yes
| Review and 2 GTM  | y
| Micro Demo (PO)   | y
| Migration script  | Done
| Tech Doc          |

### CODE REVIEW

File: upgrades/schema/Version20151006132723.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        $this->addSql('ALTER TABLE pim_notification_notification ADD comment VARCHAR(255) DEFAULT NULL');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
```
Comment (grena) [inline upgrades/schema/Version20151006132723.php]: To confirm, but I think we don't do the revert part:
```
$this->throwIrreversibleMigrationException();
```

**Comment by solivier** [comment]
plase squash the fixup comnit

**Comment by ClementGautier** [comment]
GTM after squash

**Comment by grena** [comment]
Fix the migration then GTM

**Comment by grena** [comment]
Don't forget to fill in the PR description ;)

**Pull request merged**